packages/core/src/Step.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Button from './Button'
+import Text from './Text'
+import CheckIcon from 'pcln-icons/lib/Check'
+import { getPaletteColor } from './utils'
+
+const StyledFlex = styled(Button)`
+  display: flex;
+  background-color: ${getPaletteColor('background.lightest')};
+  color: ${getPaletteColor('base')};
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+  padding: 0;
+  font-weight: unset;
+
+  &:hover {
+    background-color: ${getPaletteColor('background.lightest')};
+  }
+`
+
+function Step({ className, active, completed, children, onClick }) {
+  const color = active || completed ? 'primary' : 'text.light'
+
+  return (
+    <StyledFlex
+      className={className}
+      disabled={!onClick}
+      alignItems="center"
+      aria-selected={active}
```

Comment:
Button does not support `aria-selected`, so we need to move this to the text inside of it.
```suggestion
```